feat(user): Add followup-creating functionality.

arxanas opened the pull request:
Hi! It didn't look like the ability to post a follow-up existed, so I added it. Here's a picture of it in action:

![screen shot 2015-05-23 at 8 52 14 pm](https://cloud.githubusercontent.com/assets/454057/7786663/eb72231e-0191-11e5-9554-6afe2f938af4.png)

hfaran commented on piazza_api/network.py:
```diff
@@ -105,6 +105,40 @@ def iter_all_posts(self, limit=None):
+
+        :type  post: dict|str|int
+        :param post: Either the post dict returned by another API method, or
+            the cid` field of that post.
```
Missing a "`" before `cid`

hfaran commented on piazza_api/rpc.py:
```diff
@@ -96,6 +96,22 @@ def content_get(self, cid, nid=None):
+        """
+        r = self.request(
+            method="content.create",
+            # blank.
```
Not quite sure what this comment means?

hfaran commented:
Just a couple comments but looks good otherwise! Thanks for adding this!

arxanas commented:
Updated. (The `blank` comment was below another comment line, which I had deleted.)

I need this functionality for my own app, so I bumped the version number. Will it be retrievable from PyPI?

hfaran commented:
I actually think we can make a `0.5.0` release because there are a few other commits that have not been merged into master and released yet. So if you can actually remove the version number change, I will merge this, and then bump the version number and release separately.

arxanas commented:
Reverted. (Do you get notifications when I commit? I assume not.)

hfaran commented:
I would get notifications, but since your are committing on your own fork that I am not watching, I do not. So it's fine if you just write a comment here telling me you updated like you did just now. :)

Okay so I will merge this in now and create a release, and upload it to PyPI.

hfaran commented:
`0.5.0` is now available on PyPI. Also I added you to [CONTRIBUTORS.md](https://github.com/hfaran/piazza-api/blob/develop/CONTRIBUTORS.md); definitely feel free to add more in the future and drop your additions in here as well! :)

arxanas commented:
Great, thanks :D
